### PR TITLE
doc: fix fleet doc ref

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Fleet cannot be started with the current doc

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/content/en/docs/setup/install-fleet-manager.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Install fleet manager into the management cluster.
 
 1. Install cluster operator into the management cluster.
 
-    {{< boilerplate install-cluster-operator >}}
+    {{< boilerplate install-fleet-manager >}}
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/setup/install-fleet-manager.md:50]: is this wrong?

File: docs/content/en/docs/setup/install-fleet-manager.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Install fleet manager into the management cluster.
 
 1. Install cluster operator into the management cluster.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/setup/install-fleet-manager.md:48]: Or is this wrong?

File: docs/content/en/docs/setup/install-fleet-manager.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ kind load docker-image ghcr.io/kurator-dev/fleet-manager:{{< kurator-version >}}
 
 Install fleet manager into the management cluster.
 
-{{< boilerplate install-cluster-operator >}}
+{{< boilerplate install-fleet-manager >}}
 
 ## Install cluster operator from release package
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/setup/install-fleet-manager.md:32]: And same, fleet manager here

Comment (Xieql) [reply docs/content/en/docs/setup/install-fleet-manager.md:32]: Okay, this description also needs to be modified

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**